ramp4-pcar4/story-ramp
# Option 2: Remove `soyutils`

Closes #469

### Changes
- Removed the `soyutils.js` file.
- Removed the static CDTS files as they didn't appear to be doing anything.

### Notes
- As per the [CDTS documentation](https://cenw-wscoe.github.io/sgdc-cdts/docs/internet-docwrite-en.html), the `soyutils.js` appears to be included without any explanation as to whether its required or what it does. Perhaps the doc is wrong/outdated or theres a mystery error lurking with this option.

### Testing
1. Go to https://ramp4-pcar4.github.io/story-ramp/cdts2/index-ca-en.html#/en/00000000-0000-0000-0000-000000000000 and https://ramp4-pcar4.github.io/story-ramp/cdts2/index-ca-fr.html#/fr/00000000-0000-0000-0000-000000000000
2. Ensure that everything (including the WET template stuff) is respectful.
